bedrock/firefox/templates/firefox/campaign/unfck/emails/unfck.txt

```diff
@@ -0,0 +1,5 @@
+Tell the world what you wanna unfck
+
+------------------------------------
+
+{{ unfck_field }}
```

Comment:
I think to be safe we should use the `bleach_tags` filter here so that all HTML is stripped.

```suggestion
{{ unfck_field|bleach_tags }}
```

That filter is defined here https://github.com/mozilla/bedrock/blob/bd043fa6515d8e8a3051d1673bb9e4487147b0f8/bedrock/mozorg/templatetags/misc.py#L556-L558